code/game/gamemodes/cult/runes.dm

```diff
@@ -99,52 +99,50 @@ var/list/sacrificed = list()
 
 /obj/effect/rune/proc/convert()
 	var/list/mob/living/cultsinrange = list()
+	var/conversion_time = 900 //Default time; thirty more seconds per missing person. With three it's instant. Minimum is sixty seconds.
 	for(var/mob/living/carbon/M in src.loc)
 		if(iscultist(M))
 			continue
 		if(M.stat==2)
 			continue
 		for(var/mob/living/C in orange(1,src))
-			if(iscultist(C) && !C.stat)		//converting requires three cultists
+			if(iscultist(C) && !C.stat)
 				cultsinrange += C
+				if(conversion_time > 0) //So negative values don't botch the system
+					conversion_time -= 300 //Each person reduces conversion time by thirty seconds. Solo convert is a full minute.
 				C.say("Mah[pick("'","`")]weyh pleggh at e'ntrath!")

```

Comment:
The reason this stuff was added is because there's an automute feature if you spam the same shit too fast. Cultists were getting muted trying to summon Narsie.